### PR TITLE
[ExaquteApplication] Creation of process for refinement using divfree metric

### PR DESCRIPTION
**Pull request by riccardotosi**
Dear all,

this process calls different processes and aims at performing adaptive refinement accordingly to a given metric.
First we compute the weighted in time divergence. Then the metric (according to the divergence previously computed). Finally the external software mmg is called to compute the refinement accordingly to the given metric.

This PR should be merged after  #5078.

### CODE REVIEW

File: applications/ExaquteSandboxApplication/CMakeLists.txt [hunk 1]
```diff
@@ -6,6 +6,7 @@ message("**** configuring KratosExaquteSandboxApplication ****")
 include(pybind11Tools)
 
 include_directories( ${CMAKE_SOURCE_DIR}/kratos )
+include_directories( ${CMAKE_SOURCE_DIR}/applications/MeshingApplication )
```
Comment (philbucher) [inline applications/ExaquteSandboxApplication/CMakeLists.txt:14]: I would add [check](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/IgaApplication/CMakeLists.txt#L8-L15) if the MeshingApp is compiled to get a proper error message

Comment (riccardotosi) [reply applications/ExaquteSandboxApplication/CMakeLists.txt:14]: Done

**Comment by philbucher** [comment]
what is the status of this?

**Comment by riccardotosi** [comment]
I close it. Thanks @philbucher .

**Pull request closed without merging**